### PR TITLE
Switch powerswitch data to own metrics

### LINKED ISSUE #24: Unaligned data used for the same metric
During an island mode test I noticed that the exporter currently reads different values and uses the same prometheus metric for them. 
There are the `PowerSwitchReadings` (`rb485.*`), which generate `rctmon_grid_voltage` and `rctmon_grid_frequency`, and there are the `GridReading` (from `g_sync`), which create `rctmon_grid_voltage` and a single `rctmon_grid_frequency` (from `grid_pll`) without phase label. 
During island mode, they produce different readings (rb485 being low obviously, g_sync having the values created by the inverter. 
They should be handled differently. 

I'm willing to implement this, but I'd like to check how we want to name them. 
`rctmon_grid_*` suggests grid values, so ideally the rb485 ones, but what about people without a power switch?

I'm lacking a proper term for the g_sync values - they are about the inverter-generated, potentially grid-separated network, but household doesn't fit as well, since this a term referring to consumption from the household. 
Easiest might be to just rename the rb485 metrics `rctmon_powerswitch_*`, despite being metrics about the "real", i.e. external, grid. 

What is your opinion? 

PS: Any idea about the relevance of the `rctmon_powerstorage_frequency` metrics? They seem to be 0 for me all the time, regardless if in island mode or not.

### PR DESCRIPTION
**Pull request by sfudeus**
This fixes collision of voltage data between grid and powerstorage. Fixes #24. 
When looking into it, the easiest and most consistent way really was to just rename the prefix for the powerswitch metrics to `rctmon_powerswitch`. This decouples it from the collision in `rctmon_grid_voltage` and will not be visible for people not having a powerswitch, because rctmon_grid_frequency would not have been populated (per phase). The only change is for people with powerswitch, reading the frequency per phase, this name has changed to `rctmon_powerswitch_frequency`

### CODE REVIEW

**Comment by MichaelMMS** [comment]
Oh, I only saw your post now. Great!
I hope svalouch will merge our changes soon;)

File: src/rctmon/device_manager.py [hunk 1]
```diff
@@ -448,6 +448,18 @@ def _cb_grid(self, oid: int, value: Any) -> None:
             # g_sync.u_l_rms[2]
             elif oid == 0x2545E22D:
                 self.readings.grid.voltage_l3 = ensure_type(value, float)
+            # g_sync.u_ptp_rms[0]
```
Comment (sfudeus) [inline src/rctmon/device_manager.py:451]: fyi: These are not really related to the intended change, but I noticed they were defined and requested, but just not populated for prometheus.

**Comment by kaitimmer** [comment]
What is missing to get this merged?

I'm trying to scrape the metrics with Telefraf, which can not really handle the duplicated names.

`reading text format failed: text format parsing error in line 168: second HELP line for metric name "rctmon_grid_voltage_volt"`

Would be good to know how we can get this fixed and seeing that this PR is open for quite a while it would be good to know how we can help.

**Comment by MichaelMMS** [comment]
He hasn't done anything on github since june. Maybe he doesn't have time for it anymore.
Maybe someone should make a fork to bring this valuable project forward?

**Comment by poggenpower** [comment]
Instead of a fork @svalouch could give someone else permissions to merge into this repo. Then it all stays at the same place.
